<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerTester</class>
 <widget class="QMainWindow" name="ServerTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerTester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ButtonQuit">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>390</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAdress">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Adress :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Port :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePort">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>44201</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status :</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectStatus">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonConnect">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect !</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Call</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Afk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Acc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Chat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Online</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>add Friend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Offline</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEditCmd">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Test your own command :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCmd">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSend">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitre">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BABEL TESTER</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ServerBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelServerBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server answer browser :</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>20</width>
      <height>421</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineAdress</sender>
   <signal>returnPressed()</signal>
   <receiver>ButtonConnect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>linePort</sender>
   <signal>returnPressed()</signal>
   <receiver>ButtonConnect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditCmd</sender>
   <signal>returnPressed()</signal>
   <receiver>ButtonSend</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
